### PR TITLE
Fix/BNGP 5031 update npm packages to address latest vulnerabilities

### PR DESCRIPTION
**Pull request by Stewart-Jumbe**
Update to package.json files to address the latest vulnerabilities

https://eaflood.atlassian.net/browse/BNGP-5031

Dependabot identified vulnerabilities in the code, this branch addresses them

### CODE REVIEW

File: packages/azure-functions/Shared/reference-helpers.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const randomReferenceString = (length) => {
   const chars = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ'
   let result = ''
   for (let i = 0; i < length; i++) {
-    const randomIndex = Math.floor(randomBytes(1)[0] / 255 * chars.length)
+    const randomIndex = Math.floor(randomBytes(1)[0] / 256 * chars.length)
```
Comment (jd-barnard) [inline packages/azure-functions/Shared/reference-helpers.js:7]: Good spot Stewart! Paul suggested instead using `crypto.randomInt(0, chars.length)`. I agree that is a simpler and better approach - it should give us a more random result.

Comment (Stewart-Jumbe) [reply packages/azure-functions/Shared/reference-helpers.js:7]: Thanks John, will do

File: packages/azure-functions/package.json [hunk 2]
```diff
@@ -29,7 +29,9 @@
     "@defra/bng-geoprocessing-service": "1.85.0",
     "@defra/bng-metric-service": "1.85.0",
     "@defra/bng-utils-lib": "1.85.0",
-    "moment": "^2.29.4"
+    "moment": "^2.29.4",
+    "ip": ">=2.0.1"
```
Comment (jd-barnard) [inline packages/azure-functions/package.json:33]: why is this being added?

File: packages/geoprocessing-service/package.json [hunk 3]
```diff
@@ -21,8 +21,10 @@
   "license": "OGL-UK-3.0",
   "dependencies": {
     "@defra/bng-errors-lib": "1.85.0",
-    "gdal-async": "^3.8.3",
-    "geodesy": "^2.4.0"
+    "gdal-async": "^3.8.5",
+    "geodesy": "^2.4.0",
+    "ip": ">=2.0.1"
```
Comment (jd-barnard) [inline packages/geoprocessing-service/package.json:26]: why is this being added?

File: packages/azure-functions/package.json (hunk 2 above)
Comment (jd-barnard) [reply packages/azure-functions/package.json:33]: I think ip is a dependency of lerna and so instead perhaps it is lerna that needs upgrading?

**Review by jd-barnard — APPROVED** [review]

**Pull request merged**